dockerfiles/extract_diffs/startup_script.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+# Starts the extractDiffs command
+
+# Verify required args present
+MISSING_VAR_MESSAGE=" is required and no value was given"
+
+function testDatabaseVariables() {
+  for a in DATABASE_NAME DATABASE_HOSTNAME DATABASE_PORT DATABASE_USER DATABASE_PASSWORD
+  do
+    eval arg="$"$a
+    test $arg
+    if [ $? -ne 0 ]; then
+      echo $a $MISSING_VAR_MESSAGE
+      exit 1
+    fi
+  done
+}
+
+if test -z "$VDB_PG_CONNECT"; then
+  # Exits if the variable tests fail
+  testDatabaseVariables
+  if [ $? -ne 0 ]; then
+    exit 1
+  fi
+
+  # Construct the connection string for postgres
+  VDB_PG_CONNECT=postgresql://$DATABASE_USER:$DATABASE_PASSWORD@$DATABASE_HOSTNAME:$DATABASE_PORT/$DATABASE_NAME?sslmode=disable
+fi
+
+# Run the DB migrations
+echo "Connecting with: $VDB_PG_CONNECT"
+./goose -dir db/migrations postgres "$VDB_PG_CONNECT" up
+
+if [ $? -ne 0 ]; then
+  echo "Could not run migrations. Are the database details correct?"
+  exit 1
+fi
+
+# Run extractDiffs
+echo "Running extractDiffs..."
+./vulcanizedb extractDiffs --watchedAddresses=0x78f2c2af65126834c51822f56be0d7469d7a523e \
```

Comment:
Unfortunately, I don't think that we can add a comment for each address telling which contract is belongs to. Not sure if that's ideal, so I may play around with just using a .toml config file like we're doing with execute.